Tools/TestWebKitAPI/Tests/WebKitCocoa/SiteIsolation.mm

```diff
@@ -140,16 +222,13 @@ HTTPServer server(HTTPServer::UseCoroutines::Yes, [&](Connection connection) ->
         Util::spinRunLoop();
     EXPECT_WK_STREQ(alert.get(), "parent frame received pingpong");
 
-    __block bool done { false };
-    [webView _frames:^(_WKFrameTreeNode *mainFrame) {
-        pid_t mainFramePid = mainFrame._processIdentifier;
-        pid_t childFramePid = mainFrame.childFrames.firstObject._processIdentifier;
-        EXPECT_NE(mainFramePid, 0);
-        EXPECT_NE(childFramePid, 0);
-        EXPECT_NE(mainFramePid, childFramePid);
-        done = true;
-    }];
-    Util::run(&done);
+    checkFrameTreesInProcesses(webView.get(), {
+        { "https://example.com"_s,
+            { { RemoteFrame } }
```

Comment:
I would expect the top level main frame to be local

Reply:
It is.  This is its child.